win/rl/pynethack.cc

```diff
@@ -212,6 +212,12 @@ class Nethack
         return obs_.in_normal_game;
     }
 
+    game_end_types
+    how_done()
+    {
+        return static_cast<game_end_types>(obs_.how_done);
```

Comment:
This will be 0 (DIED) by default, even when the game hasn't finished yet. The way this is done here is that it has to be a member of `game_end_types`, which has no value for "not yet ended".